src/RNCamera.js

```diff
@@ -318,18 +325,21 @@ export default class Camera extends React.Component<PropsType, StateType> {
 
   getStatus = (): Status => {
     const { isAuthorized, isAuthorizationChecked } = this.state;
-    return isAuthorized ? 'READY' : isAuthorizationChecked ? 'PENDING_AUTHORIZATION' : 'NOT_AUTHORIZED';
-  }
+    if (isAuthorizationChecked) {
+      return CameraStatus.PENDING_AUTHORIZATION;
+    }
+    return isAuthorized ? CameraStatus.READY : CameraStatus.NOT_AUTHORIZED;
```

Comment:
I am not familiar with this logic but this is not equivalent to what what before.

Before, if `isAuthorized === true` then it would return READY regardless of the `isAuthorizationChecked`.

Is this modification a correction or a "simplification" (I also dislike oneliners with multiple checks)?